app/views/govuk_component/title.raw.html.erb

```diff
@@ -2,9 +2,9 @@
   average_title_length ||= :average
   context ||= false
 %>
-<div class="govuk-title length-<%= average_title_length %>">
+<div class="pub-c-title pub-c-title--<%= average_title_length %>">
```

Comment:
Yeah think you're right, not sure why though? 

See https://github.com/alphagov/static/pull/1117/files/71356f9b255fe913d6403bdd5006b4633b45037d#r134181601